### PR TITLE
Added translations for splash/title/website metadata in Gatsby config

### PR DESCRIPTION
**Pull request by Kurzdor**
Fixes some parts of #67 issue

### CODE REVIEW

**Comment by mihilbabin** [comment]
I think so. `uk` is the correct language code. What other choices we can have? `ua`?

**Comment by Kurzdor** [comment]
uk seems better to me.

**Comment by Kurzdor** [comment]
@ApolllonDev Fixed.

File: gatsby-config.js [hunk 1]
```diff
@@ -8,10 +8,10 @@
 
 module.exports = {
   siteMetadata: {
-    title: 'React: A JavaScript library for building user interfaces',
+    title: 'React: JavaScript-бібліотека для створення користувацьких інтерфейсів',
```
Comment (klymenkoo) [inline gatsby-config.js]:
```suggestion
    title: 'React – JavaScript-бібліотека для створення користувацьких інтерфейсів',
```

Напевно, повинно бути тире замість двокрапки?

**Pull request closed without merging**